checklists/alz_checklist.en.json

```diff
@@ -493,19 +487,17 @@
             "waf": "Security",
             "guid": "c773e7d2-6162-43a7-95a9-17e1f348ef25",
             "id": "C02.11",
-            "ammp": true,
             "severity": "High",
             "training": "https://learn.microsoft.com/learn/paths/monitor-usage-performance-availability-resources-azure-monitor/",
             "link": "https://learn.microsoft.com/azure/architecture/framework/scalability/design-capacity"
         },
         {
             "category": "Resource Organization",
             "subcategory": "Subscriptions",
-            "text": "Enforce a process for cost management",
+            "text": "[this is too vague]Enforce a process for cost management",
```

Comment:
Agree, there's been several updates around FinOps in the last 12 months, maybe something we can direct customers to there?

Reply:
I found a better link within the CAF that should probably move around.

Reply:
Also we can consolidate this with "A02.03"